BetterMD/__init__.py

```
@@ -12,7 +12,7 @@ def from_string(html:'str'):
 
     @staticmethod
     def from_file(file):
-        return Symbol.from_html(file)
+        return Symbol.from_html(file.read())
```

Ruff (0.8.2)

`Symbol` may be undefined, or defined from star imports

(F405)